lib/vis_milk2/ns-eel2/loose_eel.cpp

```diff
@@ -0,0 +1,156 @@
+#include <stdio.h>
```

Comment:
seems this isn't used, please drop it

Reply:
yep, this is to build some tool, same with eel_disasm